torch/csrc/autograd/variable.h

```diff
@@ -332,9 +332,13 @@ struct TORCH_API ViewInfo {
   /// view_fn is only saved when as_strided is not supported.
   /// If view_fn has value, we use it to recover views in backward.
   std::function<Variable(const Variable&)> view_fn_;
+  /// Analogue of view_fn but in reverse: given a view -> produce a base that is
+  /// the inverse view of view.
```

Comment:
nit: I think you mean "analogue of view_fn but in reverse: given a view -> produce the base by applying the inverse view"?